Classes/ToxWrapper/OCTToxDelegate.h

```diff
@@ -104,4 +104,12 @@
  */
 - (void)tox:(OCTTox *)tox friendAvatarUpdate:(NSData *)avatar hash:(NSData *)hash friendNumber:(int32_t)friendNumber;
 
+/**
+ * Set the callback for file send requests.
+ *
+ * @param fileName Name of file for sending
```

Comment:
Name of file for **receiving**.

This request is send by our friend when he wants to send us a file. Thus we are receiving file.